Doc/reference/grammar.rst

```diff
@@ -1,7 +1,8 @@
 Full Grammar specification
 ==========================
 
-This is the full Python grammar, as it is read by the parser generator and used
-to parse Python source files:
+This is the full Python grammar, derived directly from the grammar
```

Comment:
Question: Should we add a small note here explaining the difference between the actual grammar and the text included here?